src/models/config.ts

```diff
@@ -0,0 +1,7 @@
+export interface VoyagerConfig {
+  showDataSourceSelector?: boolean;
```

Comment:
I figure this is more of a libvoyager concern (the wrapper can check whether data is present and decide whether to set this to false). The default value for this if nothing is passed in is `true` and thus leaves the existing behaviour in the general case. I made it optional so that you can pass an empty object as a valid config. But we could make it non optional at some point and require/force 'voyager the app' to use at minimum the default config and  'libvoyager' to specify its desired bahaviour.